apps/tangle-dapp/types/restake.ts

```diff
@@ -0,0 +1,336 @@
+  /**
+   * The operator's self-bond amount.
+   */
+  readonly bond: bigint;
+
+  /**
+   * The total number of delegations to this operator.
+   */
+  readonly delegationCount: number;
+
+  /**
+   * An optional pending request to decrease the operator's self-bond,
+   * with only one allowed at any given time.
+   */
+  readonly request: OperatorBondLessRequest | null;
+
+  /**
+   * A list of all current delegations.
+   */
+  readonly delegations: Array<OperatorDelegatorBond>;
+
+  /**
+   * The current status of the operator.
+   */
+  readonly status: OperatorStatus;
+};
+
+/**
+ * A map of operator metadata keyed by the operator's account ID.
+ */
+export type OperatorMap = {
+  readonly [accountId: string]: OperatorMetadata;
+};
+
+/**
+ * Metadata of an asset, including ID, name, symbol, denomination, and status.
+ */
+export type AssetMetadata = {
+  /**
+   * The ID of the asset.
+   */
+  readonly id: string;
+
+  /**
+   * The name of the asset.
+   */
+  readonly name: string;
+
+  /**
+   * The symbol of the asset.
+   */
+  readonly symbol: string;
+
+  /**
+   * The denomination of the asset.
+   */
+  readonly decimals: number;
+
+  /**
+   * The status of the asset.
+   *
+   * @field Live - The asset is live and can be staked.
+   * @field Frozen - The asset is frozen and cannot be staked.
+   * @field Destroying - The asset is being destroyed and cannot be staked.
+   */
+  readonly status: TransformEnum<PalletAssetsAssetStatus>;
+};
+
+/**
+ * A map of asset metadata keyed by the asset's ID.
+ */
+export type AssetMap = {
+  readonly [assetId: string]: AssetMetadata;
+};
+
+/**
+ * Represents a request to unstake a specific amount of an asset.
+ * @name PalletMultiAssetDelegationDelegatorUnstakeRequest (747)
+ */
+export type DelegatorUnstakeRequest = {
+  /**
+   * The ID of the asset to be unstaked.
+   */
+  readonly assetId: string;
+
+  /**
+   * The amount of the asset to be unstaked.
+   */
+  readonly amount: bigint;
+
+  /**
+   * The round in which the unstake was requested.
+   */
+  readonly requestedRound: number;
+};
+
+/**
+ * Represents a bond between a delegator and an operator.
+ * @name PalletMultiAssetDelegationDelegatorBondInfoDelegator (749)
+ */
+export type DelegatorBondInfo = {
+  /**
+   * The account ID of the operator.
+   */
+  readonly operator: string;
+
+  /**
+   * The amount bonded.
+   */
+  readonly amount: bigint;
+
+  /**
+   * The ID of the bonded asset.
+   */
+  readonly assetId: string;
+};
+
+/**
+ * Represents a request to reduce the bonded amount of a specific asset.
+ * @name PalletMultiAssetDelegationDelegatorBondLessRequest (751)
+ */
+export type DelegatorBondLessRequest = {
+  /**
+   * The ID of the asset to reduce the bond of.
+   */
+  readonly assetId: string;
+
+  /**
+   * The amount by which to reduce the bond.
+   */
+  readonly amount: bigint;
+
+  /**
+   * The round in which the bond reduction was requested.
+   */
+  readonly requestedRound: number;
+};
```

Comment:
Same here:
`DelegatorBondInfo.amount` -> `DelegatorBondInfo.amountBonded`
`DelegatorBondInfo.assetId` -> `DelegatorBondInfo.bondedAssetId`

Also, some of them might not require comments, since their names might be already sufficient and thus commenting it could be redundant/repeating the property names: e.g. `DelegatorUnstakeRequest.assetId`

Reply:
The less comments and the more explicit property naming we have the better we may be on the long term, since each of these comments will need refactoring/maintenance when the underlying code or property changes